test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -11,9 +11,29 @@
 
     @Test
     public void isSimilar_null_returnFalse() {
-        Name testName = null; 
+        Name testName = null;
```

Comment:
avoid putting test and code change into same commit. if possible put minimal changes into a single commit